core/Pos/Util/Util.hs

```diff
@@ -189,6 +187,20 @@ instance ToJSON Byte where
 instance IsString s => MonadFail (Either s) where
     fail = Left . fromString
 
+instance Rand.DRG drg => HasLoggerName (Rand.MonadPseudoRandom drg) where
+    getLoggerName = pure mempty
```

Comment:
1. Are you sure it's needed? I see that pure toss uses named logger, for instance.
2. Why `pure mempty` instead of e. g. `pure "MonadPseudoRandom"`?

Reply:
It was needed at some point, might not be needed now (:shrug:) but maybe it'll be needed in the future again.

Reply:
> Why pure mempty instead of e. g. pure "MonadPseudoRandom"?

Done.